packages/api/src/data/datasource.ts

```diff
@@ -0,0 +1,18 @@
+import * as dotenv from 'dotenv'
+import { DataSource, Repository } from 'typeorm'
+import { Item } from '../entity/item'
+
+dotenv.config()
+
+const ItemDataSource = new DataSource({
```

Comment:
could rename this to just DataSource as we will probably store other stuff too

Reply:
![](https://media.tenor.com/3PCreHGZ3rcAAAAd/done-well.gif)